packages/platform/platform-fastify/src/components/PlatformFastify.ts

```
@@ -0,0 +1,361 @@
+    {token: PlatformApplication},
+    {token: Platform},
+    {
+      token: PlatformResponse,
+      useClass: PlatformFastifyResponse
+    },
+    {
+      token: PlatformRequest,
+      useClass: PlatformFastifyRequest
+    }
+  ];
+  private decorated: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)